src/Tool/appsettings.json

```diff
@@ -6,18 +6,18 @@
     },
     "AllowedHosts": "*",
   "DefaultPackageVersions": [
-    "Microsoft.Quantum.Compiler::0.10.2003.1102-beta",
+    "Microsoft.Quantum.Compiler::0.11.2004.1014-beta",
 
-    "Microsoft.Quantum.CsharpGeneration::0.10.2003.1102-beta",
-    "Microsoft.Quantum.Development.Kit::0.10.2003.1102-beta",
-    "Microsoft.Quantum.Simulators::0.10.2003.1102-beta",
-    "Microsoft.Quantum.Xunit::0.10.2003.1102-beta",
+    "Microsoft.Quantum.CsharpGeneration::0.11.2004.1014-beta",
+    "Microsoft.Quantum.SDK::0.11.2004.1014-beta",
```

Comment:
We'll likely want to keep this as the old Microsoft.Quantum.Development.Kit package, as Microsoft.Quantum.Sdk is an SDK package rather than a library package.

Reply:
Thanks for catching this.